packages/light-apps/src/pages/Accounts/Add/Create/StepMnemonic.tsx

```diff
@@ -37,15 +36,17 @@ export function AddAccountStepMnemonic(props: Props): React.ReactElement {
           </Button>
         </Button.Group>
       </Layout>
-      <Margin top />
-      <Header as='h3'>Copy your Mnemonic Somewhere Safe</Header>
+      <Header as='h4' wrapClass='mt3'>
```

Comment:
Question: in ui-components, there's a div wrapper around the SUIHeader, and your adding mt3 on the wrapper. Is there a reason to not just remove the div wrapper, and add `className="mt3"`?

Reply:
I think in Header it was SUI styles for `hN:first-child {margin: 0}` that are unrewritable with user-added tachyons that made me wrap it in the first place. Basically, sometimes it's easier to deal with a wrapper than with an element that has SUI styles. 

I am not 100% sure that's the way and `wrapClass` / `className` is definitely not consistent, but I guess there will be more cases when this double control is useful (ig rewriting SUI styles w tachyons, inline elements, elements that more often than not come with a wrap etc) so I want to track those a bit to come up with a sensible pattern.